serve/mlc_serve/model/model_common.py

```diff
@@ -75,13 +75,13 @@ def _is_safe_to_sample(prob_like):
     logits = torch.from_dlpack(logits)
     num_seq = len(sampling_params)
 
-    mask_random = torch.tensor(
+    mask_random_dvc = torch.tensor(
```

Comment:
`dvc` is a strange post fix. Just use `gpu` or simply `mask_random` (no post fix).

Reply:
Hello @masahi! I guess that potentially it can work on cpu in the future. I thought to rename it to `_gpu`, but it will confuse somebody in the future due to my guess. What do you think about it?

Reply:
Logits here is supposed to be always on GPU (you can add an assert). So there is no confusion.